### PR TITLE
RF | __HeaderTest >testCheckExpandMenuBuilds(), testCheckExpandMenuConfigure(), testCheckExpandMenuMyViews().

### PR DESCRIPTION
**Pull request by olpolezhaeva**
Delete unused:
 -private WebElement configure,
 -private WebElement builds
 from class BaseHeaderFooterPage.
 
Refactoring in class HeaderTest :
-testCheckExpandMenuBuilds(),
-testCheckExpandMenuConfigure(),
-testCheckExpandMenuMyViews().

### CODE REVIEW

File: src/test/java/_HeaderTest.java [hunk 1]
```diff
@@ -278,21 +278,21 @@ public void testCheckExpandMenuBuilds() {
...
     @Test
     public void testCheckExpandMenuConfigure() {
-        String buttonText = new UserBuildPage(getDriver())
+        String buttonText = new HomePage(getDriver())
                 .navigateAndClickDropDownUserMenu()
                 .clickConfigureAndGoToConfigurePage()
                 .getGen2ButtonText();
 
-        Assert.assertTrue(buttonText.contains("Add new Token"));
+        Assert.assertEquals(buttonText,"Add new Token");
```
Comment (ke-de-ve) [inline src/test/java/_HeaderTest.java:286]: add space between parameters

File: src/test/java/_HeaderTest.java [hunk 2]
```diff
@@ -278,21 +278,21 @@ public void testCheckExpandMenuBuilds() {
...
     public void testCheckExpandMenuMyViews() {
-
         String myViewsText = new HomePage(getDriver())
-                .clickMyView()
+                .navigateAndClickDropDownUserMenu()
+                .clickMyViewAndGoToMyViewPage()
                 .getTextMyView();
 
-        Assert.assertTrue(myViewsText.contains("My Views"));
+        Assert.assertEquals(myViewsText,"My Views");
```
Comment (ke-de-ve) [inline src/test/java/_HeaderTest.java:296]: add space between parameters

**Review by ke-de-ve — CHANGES_REQUESTED** [review]

**Review by ValeriiSarukhanov — APPROVED** [review]

File: src/test/java/_HeaderTest.java (hunk 2 above)
Comment (olpolezhaeva) [reply src/test/java/_HeaderTest.java:296]: Ok,Thanks!

**Review by azellus-tercia — APPROVED** [review]

**Comment by olpolezhaeva** [comment]
[RF | __HeaderTest >testCheckExpandMenuBuilds(), testCheckExpandMenuConfigure(), testCheckExpandMenuMyViews().](https://trello.com/c/3DBQjluN/1200-rf-headertest-testcheckexpandmenubuilds-testcheckexpandmenuconfigure-testcheckexpandmenumyviews)

**Review by ke-de-ve — APPROVED** [review]

**Pull request merged**